docs/source/sections/Dataloaders/scalar/implemented/visual_iced.rst

```diff
@@ -0,0 +1,16 @@
+*************************
+Visual_ice Dataloader
```

Comment:
Is it called visual_ice or visual_iced (with the d)? Maybe Martin has changed his name for it in the past few months? Either way the filename and text should be consistent.

Reply:
I just asked Martin and apparently it should be "ViSual_IceD". I guess we can just stick with "visual_iced" for readability.

Reply:
updated to visual_iced in the docs